src/main/java/seedu/us/among/logic/parser/ParserUtil.java

```diff
@@ -65,6 +62,11 @@ public static Address parseAddress(String address) throws ParseException {
         if (!Address.isValidAddress(trimmedAddress)) {
             throw new ParseException(Address.MESSAGE_CONSTRAINTS);
         }
+
+        if (!Address.isUrlValid(trimmedAddress)) {
+            trimmedAddress = "http://" + trimmedAddress;
+        }
+
```

Comment:
Is it necessary? because is Address.ValidAddress already invokes isUrlValid method within in it

Reply:
yea, keep this in here.